### DISCUSSION TITLE
Aggregated project from multiple langfuse servers

### DISCUSSION THREAD

**Discussion by juan-abia** [langfuse/langfuse#1528] [Support]
In my team use case, we are running multiple langfuse servers (one per client) as we have data security agreements. With this setup we face two main challenges:

1. We are planning on creating an aggregated project in one of the server to track the total costs (we would copy every metric except private data, that we would anonymise)
2. Each time we want to add a new model cost definition, we need to do it by hand in every server.

Our first idea is to have some SQL scripts to automate this. Is there a better way of doing it? Should we consider something else before we implement that?

**Comment by marco-introvigne** [comment]
Follow up this conversation since i'm interested too
